docs/ossm/injection/README.md

```diff
@@ -0,0 +1,100 @@
+## Installing the Sidecar
+### Injection
+In order to take advantage of all of Istio’s features, pods in the mesh must be running an Istio sidecar proxy.
+
+Automatic Istio sidecar injection can be enabled via labels on a namespace level or directly for a given pod. Automatic injection injects the proxy configuration at pod creation time using an [admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/).
+
+> **_NOTE:_** Upstream Istio offers also Manual injection using the ` istioctl kube-inject` which is not supported by OpenShift Service Mesh.
+
+After adding a correct label (see below) on a namespace, any new pods that are created in that namespace will automatically have a sidecar added to them.
+
+Automatic injection occurs at the pod-level. You won’t see any change to the deployment itself. Instead, you’ll want to check individual pods (via `oc describe`) to see the injected proxy.
+
+### Controlling the injection policy
+When a `default` control plane revision is installed or a default tag exists, it's possible to use following labels on a namespace or a pod:
+| Resource | Label | Enabled value | Disabled value |
+| --- | --- | --- | --- |
+| Namespace | `istio-injection` | `enabled` | `disabled` |
+| Pod | `sidecar.istio.io/inject` | `"true"` | `"false"` |
+
+If you are using different name for a control plane revisions, revision specific labels are instead used by a matching `istio.io/rev` label. For example, for a revision named `canary`:
```

Comment:
```suggestion
If you are using a different name for a control plane revisions, revision specific labels are instead used by a matching `istio.io/rev` label. For example, for a revision named `canary`:
```

Reply:
I think somewhere we should mention the difference between an `InPlace` `Istio` resource and a `RevisionBased` `Istio` resource and the tags they will use.